openapi/index.yaml

```diff
@@ -439,12 +678,12 @@ parameters:
     pattern: "[0-9]{4}-[0-9]{2}-[0-9]{2}"
     x-example: "2019-09-15"
 consumes:
-- application/json
+  - application/json
 produces:
-- application/json
+  - application/json
 securityDefinitions:
   SubscriptionKey:
     type: apiKey
     name: Ocp-Apim-Subscription-Key
     in: header
-    description: The API key obtained through the developer portal.
\ No newline at end of file
+    description: The API key obtained through the developer portal.
```

Comment:
```suggestion
    description: The API key obtained through the developer portal or the getService operation.
```

Reply:
Fixed in [469e173](https://github.com/pagopa/io-functions-services/pull/67/commits/469e173201722102ed04c60b05300f6618a02fe2)